packages/material-ui/src/Hidden/HiddenCss.test.js

```diff
@@ -25,116 +24,131 @@ describe('<HiddenCss />', () => {
 
   describe('the generated class names', () => {
     it('should be ok with only', () => {
-      const wrapper = shallow(
+      const { container } = render(
         <HiddenCss only="sm">
           <div className="foo" />
         </HiddenCss>,
       );
+      const root = container.firstElementChild;
```

Comment:
Why ignore non-elements?

```suggestion
      const root = container.firstChild;
```

Reply:
@oliviertassinari My bad. I've changed it to `firstChild` in all places